Retry for selecting daemon pid is not working

Retry for fetching in pid ocs-ci/tests/disruption_helpers.py in  is not working due to IndexError if pid is not fetched in first attempt. 
```
>       pid = pids[0]
E       IndexError: list index out of range
```


eg: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/7821/testReport/tests.manage.pv_services.test_ceph_daemon_kill_during_resource_creation/TestDaemonKillDuringResourceCreation/test_ceph_daemon_kill_during_resource_creation_CephFileSystem_create_pod_mon_/

--- a/tests/disruption_helpers.py
+++ b/tests/disruption_helpers.py
@@ -95,7 +95,8 @@ class Disruptions:
         # on one node. eg: More than one osd on same node.
         pids = pid.split()
         self.pids = [pid.strip() for pid in pids]
-        pid = pids[0]
+        assert self.pids, "Obtained pid value is empty."
+        pid = self.pids[0]
 
         # ret will be 0 and err will be None if command is success
         assert not any([ret, err, not pid.isdigit()]), (

